ingest/workflow/snakemake_rules/split_serotypes.smk

```diff
@@ -0,0 +1,43 @@
+"""
+This part of the workflow handles splitting the data by serotype either based on the 
+NCBI metadata or Nextclade dataset. Could use both if necessary to cross-validate.
+
+    metadata = "results/metadata.tsv"
+    sequences = "results/sequences.fasta"
+
+This will produce output files as
+
+    metadata_{serotype} = "results/metadata_{serotype}.tsv"
+    sequences_{serotype} = "results/sequences_{serotype}.fasta"
+
+Parameters are expected to be defined in `config.transform`.
+"""
+
+rule split_by_ncbi_serotype:
+  """
+  Split the data by serotype based on the NCBI metadata.
+  """
+  input:
+    metadata = "results/metadata.tsv",
+    sequences = "results/sequences.fasta"
+  output:
+    metadata = "results/metadata_{serotype}.tsv",
+    sequences = "results/sequences_{serotype}.fasta"
+  params:
+    ncbi_serotype = lambda wildcards: wildcards.serotype,
+    id_field = config["transform"]["id_field"]
+  shell:
+    """
+    if [[ "{params.ncbi_serotype}" == "all" ]]; then
+      cp {input.metadata} {output.metadata}
+      cp {input.sequences} {output.sequences}
+    else
+      augur filter \
+        --sequences {input.sequences} \
+        --metadata {input.metadata} \
+        --metadata-id-columns {params.id_field} \
+        --query "ncbi_serotype=='{params.ncbi_serotype}'" \
```

Comment:
_non-blocking_

I don't think it's necessary to thread a wildcard through `params` since we can directly access the wildcard value in the shell block
```suggestion
        --query "ncbi_serotype=='{wildcards.serotype}'" \
```

Reply:
Huh, agree! I don't know why I thought I needed to thread it through a param. Fixed in 3ddefcf046e4d2fd6c3e00eebe86ab82129d70bc